keyboards/bigseries/1key/keymaps/bbaserdem/config.h

```diff
@@ -1,9 +1,11 @@
 #ifndef CONFIG_KEYMAP_H
 #define CONFIG_KEYMAP_H
 
-#include "../../config.h"
+#ifdef TAPPING_TERM
+#undef TAPPING_TERM
+#endif
+#define TAPPING_TERM 500
 
-#define USE_SERIAL
-#define EE_HANDS
+#include "../../config.h"
```

Comment:
This isn't needed
```suggestion
```

Also, the `CONFIG_KEYMAP_H ` stuff is outdated.  It works fine, but we've switched to using `#pragma once` which accomplishes the same thing with one line vs the ifndef, define, and endif style.